pages/Crewmates.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Head from 'next/head';
+
+// Components
+import Article from '../components/article';
+
+// Images
+const Founder = '/images/1.jpg';
+
+const CrewMates = () => (
+    <div className='  bg-dark-primary text-white'>
+        <Head>
+            <title>CrewMates </title>
+
```

Comment:
Please use prettier (VS code extension) to avoid empty spaces like these and have an indentation check before making a commit.

Reply:
@shivanigupta71299 are you using Prettier now? Looks like.